common.pri

```diff
@@ -61,7 +61,7 @@ RCC_DIR = $$TEMP_PATH
 MOC_DIR = $$TEMP_PATH
 UI_DIR = $$TEMP_PATH
 
-QMAKE_LIBDIR *= $$BINARIES_PATH
+LIBPATH = $$BINARIES_PATH $$LIBPATH
```

Comment:
Why have you changed the variable name? Variable LIBPATH is deprecated; use QMAKE_LIBDIR instead